larq_compute_engine/mlir/python/graphdef_tfl_flatbuffer.cc

```diff
@@ -38,7 +40,7 @@ pybind11::bytes ConvertGraphDefToTFLiteFlatBuffer(
     const std::vector<string>& input_arrays,
     const std::vector<string>& input_dtypes,
     const std::vector<std::vector<int>>& input_shapes,
-    const std::vector<string>& output_arrays,
+    const std::vector<string>& output_arrays, const bool& should_quantize,
```

Comment:
What's the difference between `const bool& foo` and `const bool foo`?

Reply:
:+1: For primitive types we can just use `const bool foo`, no need to pass it by reference.

Reply:
:+1: Changed it to not pass it by reference.